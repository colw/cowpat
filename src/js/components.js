import React from 'react';

export const LinkExt = (props) => (
  <a {...props} target="_blank" rel="noopener"/>
)
