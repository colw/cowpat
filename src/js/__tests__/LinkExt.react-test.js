// NewsItem.react-test.js
import React from 'react';
import renderer from 'react-test-renderer';

import { LinkExt } from '../components';

test('External Link renders correctly', () => {
  const component = renderer.create(
    <LinkExt href="http://www.ruminant.press">
      <div className="headTitle">
        Ruminant
      </div>
    </LinkExt>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
