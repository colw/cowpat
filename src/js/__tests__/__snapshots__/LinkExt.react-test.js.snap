exports[`test External Link renders correctly 1`] = `
<a
  href="http://www.ruminant.press"
  rel="noopener"
  target="_blank">
  <div
    className="headTitle">
    Ruminant
  </div>
</a>
`;
